src/lib/controllers/apple-tv-controller.ts

```diff
@@ -63,29 +63,39 @@ export class AppleTvController {
         // Creates the Play/Pause switch if requested
         if (deviceConfiguration.isPlayPauseSwitchEnabled) {
             platform.logger.info(`[${deviceConfiguration.name}] Adding play/pause switch`);
-            const playPauseSwitchService = accessory.useService(Homebridge.Services.Switch, deviceConfiguration.playPauseSwitchName || 'Play', 'play-pause-switch');
-
-            // Adds the characteristics for the service
-            const onCharacteristic = playPauseSwitchService.useCharacteristic<boolean>(Homebridge.Characteristics.On);
-            onCharacteristic.valueChanged = newValue => {
-                if (onCharacteristic.value !== newValue) {
-                    platform.logger.info(`[${deviceConfiguration.name}] Play/pause switch changed to ${newValue}`);
-                    try {
-                        if (newValue) {
-                            client.play();
-                        } else {
-                            client.pause();
+            deviceConfiguration.appPlayPauseSwitches.forEach(PlayPauseSwitch => {
```

Comment:
- null/undefined check is missing. I would suggest the same pattern as with the command switches.
- Style: use for-of-loop

```
if (deviceConfiguration.appPlayPauseSwitches && deviceConfiguration.appPlayPauseSwitches.length > 0) {
    for (let appPlayPauseSwitchConfiguration of deviceConfiguration.appPlayPauseSwitches) {
        ...
    }
}
```